meson.build

```diff
@@ -57,6 +57,10 @@ if host_machine.cpu_family() == 'arm' and cpp.alignment('struct { char c; }') !=
   endif
 endif
 
+if host_machine.system() == 'windows' and cpp.get_id() != 'msvc'
+  add_project_arguments(['-Wa,-mbig-obj'], language : 'cpp')
+endif
```

Comment:
Why is this only for compilers other than MSVC? Also the check seems to assume any compiler that is not MSVC is GCC-like, not sure if this is a right assumption to make.

Reply:
Different than MSVC, but only when compiling for Windows. In my experience that usually means gcc/llvm, but that's a valid point.

I can refactor the change to add it whenever host_machine is windows, and if the compiler supports it, if that's ok with you

Reply:
Yes, please.

Reply:
Done